lib/celluloid/autostart.rb

```diff
@@ -1,5 +1,6 @@
 require "celluloid"
 
+# start up the basic Celluloid services - only necessary to do this manually in old versions
```

Comment:
Both would be nice: `celluloid/autostart` was the preferred way to call this for quite some time, so some additional clarification on the history would be good.